typings/index.d.ts

```diff
@@ -999,7 +999,7 @@ declare module 'discord.js' {
     public id: Snowflake;
     public readonly member: GuildMember | null;
     public mentions: MessageMentions;
-    public nonce: string | null;
+    public nonce: string | number | null;
```

Comment:
Q: don't you need to update message options too?

Reply:
Agree added for MessageOption and WebhookOption